Library/Homebrew/emoji.rb

```diff
@@ -0,0 +1,22 @@
+module Emoji
+  class << self
+    def tick
+      # necessary for 1.8.7 unicode handling since many installs are on 1.8.7
+      @tick ||= ["2714".hex].pack("U*")
+    end
+
+    def cross
+      # necessary for 1.8.7 unicode handling since many installs are on 1.8.7
+      @cross ||= ["2718".hex].pack("U*")
+    end
+
+    def install_badge
+      ENV["HOMEBREW_INSTALL_BADGE"] || "\xf0\x9f\x8d\xba"
+    end
+
+    def enabled?
+      !ENV["HOMEBREW_NO_EMOJI"]
+    end
+    alias_method :generic_enabled?, :enabled?
```

Comment:
I'm leaning into the pattern we've used previously in Homebrew/brew rather than what's more typical Ruby, here. I don't mind it being changed en-masse in a future PR and I'll happily follow that style then.